README.md

```diff
@@ -0,0 +1,24 @@
+
+# Notebook Images
+
+These images were created to be used with Open Data Hub (ODH) with the ODH Notebook Controller as the launcher.
+
+## Container Image Layering
+
+```mermaid
+
+flowchart TD
+
+ubi8/python-3.8 --> Base(base-ubi8-py38);
+
+```
+
+### Base Notebook Image
+
+We use the Red Hat UBI image `ubi8/python-38` [registry.access.redhat.com/ubi8/python-38](https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a) as builder image for the notebook base.
+
+Build example - UBI8 with Python 3.8:
+
+```bash
+make base-ubi8-python-3.8
+```
```

Comment:
I would reword this with the following statement:

Build the `base-ubi8-python-3.8` by running the following command:

```bash
make base-ubi8-python-3.8
```

The image will be built and pushed to the [quay.io/opendatahub/notebooks](https://quay.io/repository/opendatahub/notebooks?tab=tags) repository.

You can use a different registry by overwriting the `IMAGE_REGISTRY` variable:

```bash
make -e IMAGE_REGISRY=quay.io/YOUR_USER/notebooks base-ubi8-python-3.8
```